packages/synapse-react-client/src/components/EntityFinder/details/view/DetailsView.tsx

```diff
@@ -380,209 +703,46 @@ export function DetailsView({
-              key="id"
-              width={130}
-              dataKey="id"
-              title="ID"
-              minWidth={130}
-            />
-            {versionSelection !== VersionSelectionType.DISALLOWED && (
-              <Column<EntityFinderTableViewRowData>
-                key="version"
-                minWidth={150}
-                width={500}
-                title="Version"
-                cellRenderer={props => (
-                  <DetailsViewVersionRenderer
-                    versionSelection={versionSelection}
-                    toggleSelection={toggleSelection}
-                    {...props}
-                  />
-                )}
-                headerRenderer={
-                  <VersionColumnHeader versionSelection={versionSelection} />
-                }
-              />
-            )}
-            <Column<EntityIdAndVersionNumber>
-              key={SortBy.CREATED_ON}
-              sortable={sort != null}
-              title="Created On"
-              width={220}
-              minWidth={170}
-              cellRenderer={CreatedOnRenderer}
-            />
-            <Column<EntityIdAndVersionNumber>
-              key={SortBy.MODIFIED_ON}
-              title="Modified On"
-              width={220}
-              minWidth={170}
-              sortable={sort != null}
-              cellRenderer={ModifiedOnRenderer}
-            />
-            <Column<EntityIdAndVersionNumber>
-              key="modifiedBy"
-              title="Modified By"
-              width={500}
-              resizable
-              cellRenderer={ModifiedByRenderer}
-            />
-            {visibleTypes.includes(EntityType.FILE) && (
-              <Column<EntityFinderTableViewRowData>
-                key={'SIZE'}
-                title="Size"
-                width={200}
-                minWidth={85}
-                sortable={false}
-                resizable={true}
-                cellRenderer={SizeRenderer}
-              />
-            )}
-            {visibleTypes.includes(EntityType.FILE) && (
-              <Column<EntityFinderTableViewRowData>
-                key={'MD5'}
-                title="MD5"
-                width={200}
-                sortable={false}
-                resizable={true}
-                cellRenderer={MD5Renderer}
-              />
-            )}
-            {visibleTypes.includes(EntityType.FILE) && (
-              <Column<EntityFinderTableViewRowData>
-                key={'addToDownloadCart'}
-                title=""
-                width={40}
-                minWidth={40}
-                sortable={false}
-                resizable={true}
-                cellRenderer={AddFileToDownloadListRenderer}
-              />
-            )}
-          </BaseTable>
-        )}
-      </AutoResizer>
+            },
+          }}
+        />
+      )}
+      {isLoading && (
+        <div className="EntityFinderDetailsViewPlaceholder">
+          <SynapseSpinner size={30} />
```

Comment:
Would a skeleton be preferred in this case?

Reply:
I think skeletons are always better than spinners! In this case, the editor area is already rendered with a defined height/width, so a skeleton wouldn't affect layout shift. So I don't think it's necessarily worth the effort to change it